packages/accordion/src/useAccordion.ts

```diff
@@ -5,49 +5,84 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import { useState } from 'react';
+import { useState, HTMLProps } from 'react';
 import {
   composeEventHandlers,
   generateId,
   KEY_CODES,
   getControlledValue
 } from '@zendeskgarden/container-utilities';
 
+export interface IUseAccordionProps {
+  idPrefix?: string;
+  expandedSections?: number[];
+  onChange?: (expanded: number[]) => any;
+  expandable?: boolean;
+  collapsible?: boolean;
+}
+
+interface IHeaderProps extends HTMLProps<any> {
+  ariaLevel?: number | null;
+  role?: any;
+}
+
+interface ITriggerProps extends HTMLProps<any> {
+  index?: number;
+  role?: any;
+  tabIndex?: any;
+}
+
+interface IPanelProps extends HTMLProps<any> {
+  index?: number;
+  role?: any;
```

Comment:
Is `role` a valid member of `HTMLProps<any>` already?

Reply:
Yeah, as long as we don't pass `null` in as the value for `role`, but [we do](https://github.com/zendeskgarden/react-containers/blob/master/packages/accordion/stories.js#L50). Is there another way around this? Is this an indication that TypeScript is telling us we're doing something weird?